Trimble.Modus.Components/Controls/DropDown/TMDropDown.xaml.cs

```diff
@@ -97,27 +94,17 @@ private void OnSelected(object sender, SelectedItemChangedEventArgs e)
             UpdateCellColor((ListView)sender);
         }
         label.Text = e.SelectedItem.ToString();
-        if (count > 0 && PopupService.Instance.PopupStack.Count > 0)
+        if (PopupService.Instance.PopupStack.Count > 0)
         {
-
             PopupService.Instance?.DismissAsync();
             Close();
         }
         SelectedIndex = e.SelectedItemIndex;
-        count = 1;
     }
 
     private void OnTapped(object sender, EventArgs e)
     {
-        isVisible = !isVisible;
```

Comment:
Don't we need this?

Reply:
We do not need this because if we click outside of the popup, popup will close. Hence we dont need this